### ISSUE TITLE
Vertical tabs bar inner border issue

### ISSUE THREAD

**Issue by andrewberty** [brave/brave-browser#34450]
## Description 
Vertical tab bar has white inner border. Always rendered while collapsed, Flicker rendering depending on width while expanded

## Steps to Reproduce
   1. Enable vertical tabs 
   2. Disable show title bar
   3. Disable expand on mouse hover

## Actual result:

https://github.com/brave/brave-browser/assets/95014217/d95023e7-4744-4ab6-a599-84daab56cc07

## Brave version (brave://version info)
Brave | 1.60.118 Chromium: 119.0.6045.163 (Official Build) (64-bit)
Revision | b0599b790dbdfe28428202afe2b54758a044ccfc
OS | Windows 11 Version 23H2 (Build 22631.2715)

**Comment by is-this-c** [comment]
This glitchy white line exists with light or dark themes, and with left- or right-alignment, and with or without a scroll bar, and its width is variable.  This issue seems duplicated in #30764 and #35321.

It seems to have existed for a while.  @sangwoo108 Your contributions to vertical tabs are great. But would you be able to fix this issue?  This glitchy white line is jarring.  I think that it should be removed completely.  Further, if vertical tab scroll bar background color is changed to vertical tab side bar background color, I think that it would look more minimal and very pleasing like Edge's.

**Comment by sangwoo108** [comment]
Thanks for pinging me. It's time for me to have a new Win11 machine 😄

**Comment by sangwoo108** [comment]
Hello, everyone. I found that this isn't happening on Nightly. @zenparsing I'm not sure but did we fix this recently? I'm curious if we're going to uplift the fix

**Comment by andrew-george** [comment]
Hello @sangwoo108,
I tried nightly just now and unfortunately it's the same to me

![explorer_GyTmtXl4Qi](https://github.com/brave/brave-browser/assets/95014217/3cf8f4e6-506a-4ca7-bd66-66f0982dc4fc)

**Comment by sangwoo108** [comment]
Thanks @endrew. Could you share your DPI? like 150% 175% from the display settings.

**Comment by andrew-george** [comment]
Wow, interesting. I think that's it. 
That border only appears at 125% 175% 225% but doesn't appear at 100% 150% 200%

**Comment by sangwoo108** [comment]
Yeah odd number usually affects that Thanks for checking. I'll take a look soonish

**Comment by bsclifton** [comment]
This should be fixed now - I can't reproduce anymore
